### DISCUSSION TITLE
What is the sum of all positive integers less than 1000 that are divisible by both 3 and 5 ?

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#28] [Galaxy Qs] [answered]
What is the sum of all positive integers less than 1000 that are divisible by both 3 and 5 ?

**Comment by code-alt** [comment, marked as answer]
Hmm, maybe 33165?

**Reply by Sriansh-raj** [reply]
Correct 🎉 
congratulations @code-alt you just earned a prestigious `galaxy Brain` achievement.
😄

**Reply by Sriansh-raj** [reply]
@code-alt pls give some time and make a pr and enter your name in [Users_who_got_Galaxy_Brain.md](https://github.com/Sriansh-raj/GitHub-Achievements/blob/main/Users_who_got_Galaxy_Brain.md) to make a history 😄
